lapce-core/src/word.rs

```diff
@@ -428,7 +428,7 @@ impl<'a> WordCursor<'a> {
             if matching_pair_direction(c) == Some(true) {
                 let opening_bracket_offset = self.inner.pos();
                 if let Some(closing_bracket_offset) = self.match_pairs() {
-                    if (opening_bracket_offset..closing_bracket_offset)
+                    if (opening_bracket_offset..=closing_bracket_offset)
```

Comment:
This fixes a bug where the offset immediately before the closing bracket (i.e.: `(   |)`) was not considered enclosed.